specs/Standards/MultiToken/Core.md

```diff
@@ -0,0 +1,355 @@
+
+## Motivation
+
+
+In the three years since [ERC-1155] was ratified by the Ethereum Community, Multi Token based contracts have proven themselves valuable assets. Many blockchain projects emulate this standard for representing multiple token assets classes in a single contract. The ability to reduce transaction overhead for marketplaces, video games, daos, and exchanges is appealing to the blockchain ecosystem and simplifies transactions for developers. 
+
+Having a single contract represent NFTs, FTs,  and tokens that sit inbetween greatly improves efficiency. The standard also introdued the ability to make batch requests with multiple asset classes reducing complexity. This standard allows operations that currently require _many_ transactions to be completed in a single transaction that can transfer not only NFTs and FTs, but any tokens that are a part of same token contract.
+
+With this standard, we have sought to take advantage of the ability of the NEAR bockchain to scale. It's sharded runtime, and [storage staking] model that decouples [gas] fees from storage demand, enables ultra low transaction fees and greater on chain storage ( see [Metadata]extenson).   
+
+With the aformentioned, it is note worthy to mention that like the [NFT] standard the Multi Token standard, implements `mt_transfer_call`,
+which allows, a user to attach many tokens to a call to a separate contract. Additionally this standard includes an optional [Approval Management] extension. The extension allows marketplaces to trade on behalf of a user, providing additional flexibility for dApps.
+
+Prior art:
+
+- [ERC-721]
+- [EIP-1155 for multi-tokens](https://eips.ethereum.org/EIPS/eip-1155)
+- [NEAR's Fungible Token Standard](../FungibleToken/Core.md), which first pioneered the "transfer and call" technique
+- [NEAR's Non-Fungible Token Standard](../NearFungibleToken/Core.md)
+
+## Reference-level explanation
+
+**NOTES**:
+- All amounts, balances and allowance are limited by `U128` (max value `2**128 - 1`).
+- Token standard uses JSON for serialization of arguments and results.
+- Amounts in arguments and results have are serialized as Base-10 strings, e.g. `"100"`. This is done to avoid JSON limitation of max integer value of `2**53`.
+- The contract must track the change in storage when adding to and removing from collections. This is not included in this core multi token standard but instead in the [Storage Standard](../StorageManagement.md). 
+- To prevent the deployed contract from being modified or deleted, it should not have any access keys on its account.
+
+### MT Interface
+
+```ts
+// The base structure that will be returned for a token. If contract is using
+// extensions such as Approval Management, Enumeration, Metadata, or other
+// attributes may be included in this structure.
+type Token = {
+  id: string,
+  owner_id: string | null
```

Comment:
It's because you may not have an owner for a fungible style token in the contract. Ie. there isn't one owner but potentially 100s/1000s . So owner_id is is just optional to signal that this token entity can be owned by one person.